content/index.txt

```diff
@@ -4,14 +4,54 @@ view: home
 ----
 subtitle: We empower communities to work together and build innovative technology for the public good.
 ----
-tagline: Supporting Collaborative Communities
+tagline: Strategy & Fiscal Sponsorship
 ----
-description: We are a 501(c)(3) nonprofit supporting open source projects and open collaboration in public interest technology. We envision a future of technology where value-driven communities have tools that reflect their values. By connecting people across domains and building shared resources, we are working towards that future.
+description: We are a US-based 501(c)(3) nonprofit supporting open collaboration in public interest technology through fiscal sponsorship, collaborative projects in open infrastructure, and consulting. We envision a future where communities build technology infrastructure that reflect their values.
 ----
 background: /assets/background.jpg
 
 ----
 
+programs:
+communities:
```

Comment:
indents matter here. needs to have 2 spaces indent after programs.

Reply:
k